snomed/com.b2international.snowowl.validation.snomed/src/main/resources/scripts/rule_duplicate_members.groovy

```diff
@@ -0,0 +1,65 @@
+package scripts
+
+import com.b2international.index.aggregations.Aggregation
+import com.b2international.index.aggregations.AggregationBuilder
+import com.b2international.index.query.Expressions
+import com.b2international.index.query.Expressions.ExpressionBuilder
+import com.b2international.index.revision.RevisionSearcher
+import com.b2international.snowowl.core.ComponentIdentifier
+import com.b2international.snowowl.core.date.EffectiveTimes
+import com.b2international.snowowl.core.terminology.ComponentCategory
+import com.b2international.snowowl.snomed.cis.SnomedIdentifiers
+import com.b2international.snowowl.snomed.common.SnomedTerminologyComponentConstants
+import com.b2international.snowowl.snomed.datastore.index.entry.SnomedRefSetMemberIndexEntry
+import com.b2international.snowowl.snomed.snomedrefset.SnomedRefSetType
+import com.google.common.collect.Lists
+
+final RevisionSearcher searcher = ctx.service(RevisionSearcher.class)
+
+final List<ComponentIdentifier> issues = Lists.newArrayList()
+
+final String script = String.format("return %s + '_' + %s",
+					"doc.referenceSetId.value",
+					"doc.referencedComponentId.value")
+
+final ExpressionBuilder queryBuilder = Expressions.builder().filter(SnomedRefSetMemberIndexEntry.Expressions.refSetTypes(Collections.singleton(SnomedRefSetType.SIMPLE)))
```

Comment:
The candidates should be restricted to active members only, we don't leave any way of fixing the issue otherwise (except mass whitelisting) for released/historical components.